### PR TITLE
Add one sentence defining Type III CRISPR

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

Add a definition of Type III CRISPR, per @nilswellhausen comment
Feel free to edit or suggest alternatives!

### Suggested reviewers (optional)

@nilswellhausen @SystemsResearch @jessegmeyerlab

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Review by SystemsResearch — CHANGES_REQUESTED** [review]
I think the sentence is fine.

I also had a read through the whole section again and checked the references. Two of the preprints have now been published:

- 10.1101/2020.10.14.20212670 on line 201 has been published as 10.1016/j.xcrm.2021.100319
- 10.1101/2020.09.13.20193581 on line 219 has been published as 10.1016/j.bios.2021.113207

Should they be updated?

**Review by RLordan — APPROVED** [review]

File: content/10.diagnostics.md [hunk 1]
```diff
@@ -197,10 +197,10 @@ Combining this ITP purification with loop-mediated isothermal amplification, the
 All these methods required upstream nucleic acid amplification prior to CRISPR-based detection. 
 They are relying on type V (Cas12-based) and type IV (Cas13-based) CRISPR systems.
 In contrast, type III CRISPR systems have the unique property of initiating a signaling cascade, which could boost the sensitivity of direct RNA detection. 
-<!--To Do: It would be great to add just one sentence on what the type III CRISPR system is otherwise its confusing why/how a signaling cascade is initiated if one does not know what the differences are. (from Nils)-->
+In type III CRISPR systems, guide crRNAs are bound by several Cas proteins [@doi:10.1007/978-1-4939-2687-9_4] and can target both DNA and RNA molecules [@doi:10.1016/j.mib.2017.08.003; @doi:10.1016/j.tim.2016.09.012].
 A study tested this hypothesis using the type III-A CRISPR RNA (crRNA)-guided surveillance complex from Thermus thermophilus [@doi:10.1101/2020.10.14.20212670].
```
Comment (rando2) [inline content/10.diagnostics.md:201]: Update to published doi? 10.1016/j.xcrm.2021.100319

**Comment by SystemsResearch** [comment]
@rando2 I agree this is a sensible solution: we cite the published version and just remove the statement that has disappeared in that version.

File: content/10.diagnostics.md (hunk 1 above)
Comment (rando2) [reply content/10.diagnostics.md:201]:
```suggestion
A study tested this hypothesis using the type III-A CRISPR RNA (crRNA)-guided surveillance complex from _Thermus thermophilus_ [@doi:10.1016/j.xcrm.2021.100319]. 
```

File: content/10.diagnostics.md [hunk 2]
```diff
@@ -197,10 +197,10 @@ Combining this ITP purification with loop-mediated isothermal amplification, the
...
 They are relying on type V (Cas12-based) and type IV (Cas13-based) CRISPR systems.
 In contrast, type III CRISPR systems have the unique property of initiating a signaling cascade, which could boost the sensitivity of direct RNA detection. 
-<!--To Do: It would be great to add just one sentence on what the type III CRISPR system is otherwise its confusing why/how a signaling cascade is initiated if one does not know what the differences are. (from Nils)-->
+In type III CRISPR systems, guide crRNAs are bound by several Cas proteins [@doi:10.1007/978-1-4939-2687-9_4] and can target both DNA and RNA molecules [@doi:10.1016/j.mib.2017.08.003; @doi:10.1016/j.tim.2016.09.012].
 A study tested this hypothesis using the type III-A CRISPR RNA (crRNA)-guided surveillance complex from Thermus thermophilus [@doi:10.1101/2020.10.14.20212670]. 
 They showed that activation of the Cas10 polymerase generates three products (cyclic nucleotides, protons, pyrophosphates) that can all be used to detect SARS-CoV-2 RNA. 
-Direct detection was 100-fold more sensitive than direct detection by Cas13. 
+Direct detection was 100-fold more sensitive than direct detection by Cas13.
```
[the remark this answers is not in the document]
Comment (rando2) [reply content/10.diagnostics.md:203]:
```suggestion
```

**Pull request merged**